server.js

```diff
@@ -14,124 +14,59 @@ const { parse } = require('url')
 const next = require('next')
 
 const { getItemByPath } = require('./src/utils/sidebar')
+const { getRedirect } = require('./src/redirects.js')
```

Comment:
Should this live in src/utils @fabiosantoscode @shcheklein ? Thanks

Reply:
yep, we can move them .. move redirects-list to the root porbably

Reply:
Will move and PR

Reply:
Thanks!